src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -173,7 +184,8 @@ the MIT License. See LICENSE in the project root for license information. -->
 
             <!-- Additional Colors (Right Column) -->
             <ItemsControl Grid.Row="1"
-                          Grid.Column="1">
+                          Grid.Column="1"
+                          Style="{StaticResource ItemsControlStyle}">
                 <StackPanel Style="{StaticResource ColorStackPanelStyle}">
                     <TextBlock x:Uid="ColorScheme_Foreground"
                                Style="{StaticResource ColorHeaderStyle}"/>

```

Comment:
Fix the `ColorHeaderStyle` to have the correct font size (style located at top of page).

Reply:
I'd take a look at the other styles too. Start by making the left column (color table) look right, then update the right column (additional colors) to match that styling.

I _think_ if you just mess with the styles above, everything should just update to match. But this page is wonky so be careful.

Reply:
`ColorHeaderStyle` doesn't have the font size set. I don't know what other styling you're looking for on this page.